src/daemon/rpmostree-sysroot-upgrader.c

```diff
@@ -1160,13 +1121,13 @@ get_base_commit_for_deployment (RpmOstreeSysrootUpgrader  *self,
                                 char                     **out_base,
                                 GError                   **error)
 {
-  GKeyFile *origin = ostree_deployment_get_origin (deployment);
-  gboolean is_local;
+  g_autoptr(RpmOstreeOrigin) origin = NULL;
 
-  if (!_rpmostree_origin_is_locally_assembled (origin, &is_local, error))
+  origin = rpmostree_origin_parse_deployment (deployment, error);
+  if (!origin)
     return FALSE;
 
-  if (is_local)
+  if (rpmostree_origin_is_locally_assembled (origin))
```

Comment:
We need to make sure we'll follow the same convention here for local initramfs of making the local assembly a child of the base commit.